src/neo/SmartContract/Native/Tokens/NeoToken.cs

```diff
@@ -205,13 +186,13 @@ private bool Vote(StoreView snapshot, UInt160 account, ECPoint voteTo)
             return true;
         }
 
-        [ContractMethod(1_00000000, ContractParameterType.Array, CallFlags.AllowStates)]
-        private StackItem GetCandidates(ApplicationEngine engine, Array args)
+        [ContractMethod(1_00000000, CallFlags.AllowStates)]
+        public (ECPoint PublicKey, BigInteger Votes)[] GetCandidates(StoreView snapshot)
```

Comment:
This is converted as an `Array`? in order to simplify, maybe we need two syscalls one for votes and other for public keys

Reply:
It is converted as an array of structs. This result is the same as the original.